wordvecspace/tornado_wvs.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import tornado.ioloop
+import tornado.web
+
+from kwikapi.tornado import RequestHandler
+from kwikapi import API
+from logging import Logger
+
+from typing import Union
+
+from wordvecspace import WordVecSpace
+
+class Tornado_wvs(WordVecSpace):
```

Comment:
@RamanjaneyuluIdavalapati class name doesn't meet PEP8 guidelines.

Reply:
@RamanjaneyuluIdavalapati Let us prefer "composition" over "inheritance". This means that instead of inheriting from `WordVecSpace` we ought to create an instance of that and then use it.